backend/src/api/events.py

```
@@ -171,6 +183,133 @@ async def get_event_dashboard_stats(
+        )
+
+
+@router.post(
+    "/conflicts/resolve",
+    response_model=ConflictResolveResponse,
+    summary="Resolve a conflict group",
+    description="Batch-update attendance for events in a conflict group",
+)
+async def resolve_conflict(
+    request: ConflictResolveRequest,
+    ctx: TenantContext = Depends(require_auth),
```

Ruff (0.15.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)